Dockerfile

```diff
@@ -0,0 +1,29 @@
+#Deriving the latest base image
+FROM python:latest
+
+
+#Labels as key value pair
+LABEL Maintainer="gpa.me17"
+
+
+# Any working directory can be chosen as per choice like '/' or '/home' etc
+# i have chosen /usr/app/src
+WORKDIR /usr/app
+
+#to COPY the remote file at working directory in container
+COPY entry.py ./
+COPY src ./src
+COPY data ./data
+COPY results ./result
+COPY test ./test
```

Comment:
Would this be equal to simply doing: 
`COPY .  .`

Reply:
yes. fixed